3scale/porta
# THREESCALE-8319: Fix API update of  account_approval_required attribute in Usage Rules

**What this PR does / why we need it**:

Makes `account_approval_required` editable through API.

Also, fixes another issues along the way, that made the checkbox always disabled.

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-8319

**Verification steps** 

Call the API and verify that the setting is updated to `true`/`false` accordingly:

```
curl -X 'PUT' \
  'http://provider-admin.3scale.localhost:3000/admin/api/settings.json' \
  -H 'Content-Type: application/x-www-form-urlencoded' \
  -d 'account_approval_required=false&access_token=whatever'
```

**Special notes for your reviewer**:

This is draft, because it misses tests.
